plugins/ag-charts-generate-example-files/src/executors/generate/generator/utils/frameworkFilesGenerator.ts

```diff
@@ -290,7 +292,7 @@ export const frameworkFilesGenerator: Record<InternalFramework, ConfigGenerator>
             },
             boilerPlateFiles,
             // Other files, not including entry file
-            scriptFiles: Object.keys(otherScriptFiles),
+            // scriptFiles: Object.keys(otherScriptFiles),
```

Comment:
?

Reply:
Not needed (anymore) - in the same way its not needed for angular